stdlib/public/Concurrency/Actor.cpp

```diff
@@ -817,6 +817,30 @@ class alignas(sizeof(void *) * 2) ActiveActorStatus {
     return offsetof(ActiveActorStatus, DrainLock);
   }
 #endif
+
+  void traceStateChanged(HeapObject *actor) {
+    // Convert our state to a consistent raw value. These values currently match
+    // the enum values, but this explicit conversion provides room for change.
+    uint8_t traceState = 255;
+    switch (getActorState()) {
+    case ActiveActorStatus::Idle:
```

Comment:
Should these values be exported in a private header somewhere for Instruments to slurp up? 

And maybe we have a version number as well?

Reply:
Or do we think we'd just change the signpost ID value if we had a breaking change?

Reply:
So I think the best way to handle this is to add the new values by appending to the log string. Parsers should be smart enough that the old versions ignore anything new and new version know to ignore the old fields and look at the new fields. If we do have to "deprecate" a field, we would ideally keep it in a good enough state for 1 year before we completely removed it. Any changes that remove from the format would need a 1 year lead time.